packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -210,12 +210,8 @@ class Navigator extends StatefulWidget {
   /// The route name will be passed to that navigator's [onGenerateRoute]
   /// callback. The returned route will be pushed into the navigator. The set of
   /// most valuable keys will be used to construct an appropriate [Hero] transition.
-  ///
-  /// Uses [openTransaction()]. Only one transaction will be executed per frame.
   static void pushNamed(BuildContext context, String routeName, { Set<Key> mostValuableKeys }) {
-    openTransaction(context, (NavigatorTransaction transaction) {
-      transaction.pushNamed(routeName, mostValuableKeys: mostValuableKeys);
-    });
+    of(context).pushNamed(routeName, mostValuableKeys: mostValuableKeys);
```

Comment:
for clarity and for copy-paste sanity, always refer to `Navigator.of` even within `Navigator`.